docs/Reference/API-Methods.md

```diff
@@ -17,32 +17,119 @@ Content-Type: application/json
 
 **Request Body**
 
-`payload` : *string* - Base64 encoded payload
+`addresses` : *array of strings* - Orion node keys to include in the privacy group
 
-`from` : *string*  - Public key of sender
+`from` : *string* - Orion node key of node creating the privacy group
 
-`to` : *array of strings* - List of public keys to receive this payload
+`name` : *string* - Name of the privacy group  
 
+`description` : *string* - Description for the privacy group
 
 **Returns**
 
-`key` : *string* - Key used to receive the payload
+`privacy group` : *object* - Privacy group object 
 
 !!! example
     ```bash tab="curl HTTP request"
-    curl -X POST http://127.0.0.1:8888/send \
+    curl -X POST http://127.0.0.1:8888/createPrivacyGroup \
+      -H 'Content-Type: application/json' \
+      -d '{ 
+        "addresses": [ 
+          "g59BmTeJIn7HIcnq8VQWgyh/pDbvbt2eyP0Ii60aDDw=", 
+          "negmDcN2P4ODpqn/6WkJ02zT/0w0bjhGpkZ8UP6vARk=" 
+        ], 
+        "from": "negmDcN2P4ODpqn/6WkJ02zT/0w0bjhGpkZ8UP6vARk=", 
+       "name": "Organisation A", 
+       "description": "Contains members of Organisation A" 
+     }'
+    ```
+   
+    ```json tab="Result"
+    {
+        "privacyGroupId": "68/Cq0mVjB8FbXDLE1tbDRAvD/srluIok137uFOaClOIn4m3Nn0P4jDuRGhWK3AAVdvkqQ==",
+        "name": "Organisation A",
+        "description": "Contains members of Organisation A",
+        "type": "PANTHEON"
+    } 
+    ```
+
+## deletePrivacyGroup
+
+Deletes a privacy group.
+
+**HTTP Verb**
+POST 
+
+**Headers:**
+Content-Type: application/json
+
+**Request Body**
+
+`privacyGroupId` : *string* - ID of the privacy group to delete
+
+`from` : *string* - Orion node key of node deleting the privacy group
+
+**Returns**
+
+`privacyGroupId` : *string* - ID of the deleted privacy group 
+
+!!! example
+    ```bash tab="curl HTTP request"
+    curl -X POST http://127.0.0.1:8888/deletePrivacyGroup \
       -H 'Content-Type: application/json' \
       -d '{
-    	"payload": "SGVsbG8sIFdvcmxkIQ==",
-    	"from": "4xanJzyaDPcBVMUSwl/tLp+DbXzd3jF9MKk1yJuyewE=",
-    	"to": ["YE5cJRJYTRO4XFo7yuAi/0K9DwjySGjsHB2YrFPnJXo="]
-    }'
+        "privacyGroupId": "68/Cq0mVjB8FbXDLE1tbDRAvD/srluIok137uFOaClOIn4m3Nn0P4jDuRGhWK3AAVdvkqQ==",
+        "from": "negmDcN2P4ODpqn/6WkJ02zT/0w0bjhGpkZ8UP6vARk="
+      }'
     ```
    
     ```json tab="Result"
-    {"key":"wS+RMprLKIuCaHzOBfPeHmkJWUdOJ7Ji/9U3qj2jbXQ="}
+    "68/Cq0mVjB8FbXDLE1tbDRAvD/srluIok137uFOaClOIn4m3Nn0P4jDuRGhWK3AAVdvkqQ==" 
```

Comment:
Is this an example of a privacy group id? It is the wrong length. This is the correct length `wS+RMprLKIuCaHzOBfPeHmkJWUdOJ7Ji/9U3qj2jbXQ=`

Reply:
Fixed